### PR TITLE
feat(payment): PAYPAL-876 bump checkout-sdk

### PR DESCRIPTION
**Pull request by andriiVitvitskyi1990**
## What?
bump checkout-sdk
## Why?
According to task PAYPAL-876

## Testing / Proof
...

@bigcommerce/checkout @davidchin

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bigcommerce/checkout",
-  "version": "1.112.0",
+  "version": "1.110.1",
```
Comment (BC-ACherednichenko) [inline package.json:3]: @andriiVitvitskyi1990 checkout version is fully managed by circle, no need  to modify

File: package-lock.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bigcommerce/checkout",
-  "version": "1.112.0",
+  "version": "1.110.1",
```
Comment (BC-ACherednichenko) [inline package-lock.json:3]: @andriiVitvitskyi1990 checkout version is fully managed by circle, no need  to modify

**Review by BC-ACherednichenko — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 2 above)
Comment (andriiVitvitskyi1990) [reply package-lock.json]: fixed

**Review by BC-ACherednichenko — APPROVED** [review]

**Comment by capsula4** [comment]
superseded by https://github.com/bigcommerce/checkout-js/pull/509

**Pull request closed without merging**